### DISCUSSION TITLE
Simple Mouthpiece

### DISCUSSION THREAD

**Discussion by stardusthamster** [cardonabits/haxo-hw#55] [Show and tell]
As I am a hobby flute player it is not important for me to have a real saxophone mouthpiece. Nevertheless I was looking for different options and found some at Thomann:
- https://www.thomann.de/de/bari_esprit_alto_saxophone.htm - complete moutpiece for 19 EUR
- https://www.thomann.de/de/jupiter_try_out_box.htm - a try out box with different mouthpieces for 18 EUR, one a complete saxophone moutpiece
I never bought one of those though.

But as I said I am not familiar with the sax mouthpieces so I looked for something else:
I found a bottled tea for kids with a sort of mouthpiece. I disassembled it and took the white inner piece.
When I tried to fit it on my haxophone it just stuck without any additional effort!

So now I have a smaller mouthpiece for the haxophone that is easier to play. 

Will have to buy some more of this fruit tea to have some spare moutpieces... 

![fruit_teat_bottle](https://github.com/cardonabits/haxo-hw/assets/154882810/7c72d534-7ed6-4803-9952-5d11a7a46c7e)
![cap_open_2](https://github.com/cardonabits/haxo-hw/assets/154882810/d3a4dc8a-a1fc-456c-8aef-e09d6264c307)
![disassembled_cap](https://github.com/cardonabits/haxo-hw/assets/154882810/16dc14b8-7213-41aa-9bb4-1bce42109710)
![haxophone_with_moutpiece](https://github.com/cardonabits/haxo-hw/assets/154882810/45e3534e-44c2-4f5e-ac75-dfcfff7f4e45)

**Comment by jcard0na** [comment]
Scrappy and creative... love it!
Maybe you can try dipping the vent tube into the bottle to dispense sips of fruit tea while playing 🤣.

Thanks for sharing.
